src/components/UserPortal/ChatRoom/ChatRoom.spec.tsx

```
@@ -23,6 +23,39 @@ vi.mock('react-bootstrap', async () => {
+      toggleGroupChatDetailsModal,
+      _groupChatDetailsModalisOpen,
+      chatRefetch,
+    }: {
+      toggleGroupChatDetailsModal: () => void;
+      _groupChatDetailsModalisOpen?: boolean;
+      chatRefetch: () => Promise<unknown>;
+    }) => {
+      _capturedChatRefetch = chatRefetch;
+      return (
+        <div data-testid="groupChatDetailsModal">
+          <button aria-label="close" onClick={toggleGroupChatDetailsModal}>
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)